scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -946,6 +946,30 @@ def test_magnitude_single_rotation():
     assert_allclose(result2, 0)
 
 
+def test_approx_equal():
+    rng = np.random.RandomState(0)
+    p = Rotation.random(10, random_state=rng)
+    q = Rotation.random(10, random_state=rng)
+    r = p * q.inv()
+    r_mag = r.magnitude()
+    atol = np.median(r_mag)  # ensure we get mix of Trues and Falses
+    assert np.all(p.approx_equal(q, atol) == ((0 <= r_mag) & (r_mag < atol)))
```

Comment:
Why check `0 <= r_mag`?

Also possibly better to use `assert_equal` (it seems to be the preferred style or at least was). But probably doesn't matter.